test/rc-test.js

```diff
@@ -42,15 +42,18 @@ test('custom config and aliases', function (t) {
     t.equal(rc.path, rc.p, 'path alias')
     t.equal(rc.preinstall, 'somescript.js', 'correct script')
     t.equal(rc.preinstall, rc.i, 'preinstall alias')
-    t.deepEqual(rc.prebuild, rc.pb, 'prebuild alias')
+    t.equal(rc.target, 'X.Y.Z', 'correct target')
+    t.equal(rc.target, rc.t, 'target alias')
+    t.equal(rc.runtime, 'electron', 'correct runtime')
+    t.equal(rc.runtime, rc.r, 'runtime alias')
     t.end()
   })
 })
 
 test('using --all will build for all targets', function (t) {
   var args = [
-    '--prebuild vX.Y.Z',
-    '--prebuild vZ.Y.X',
+    '--prebuild X.Y.Z',
+    '--prebuild Z.Y.X',
```

Comment:
The leading `v` is now supported: https://github.com/lgeiger/node-abi/commit/4980cb30e46f246f5ef4b4a01cac426a4a3e3ffe